consensus/consortium/v2/consortium.go

```diff
@@ -203,6 +207,51 @@ func (c *Consortium) Author(header *types.Header) (common.Address, error) {
 	return header.Coinbase, nil
 }
 
+// VerifyBlobHeader verifies the header of a blob block
+func (c *Consortium) VerifyBlobHeader(block *types.Block, blobs []kzg4844.Blob, proofs []kzg4844.Proof, commitments []kzg4844.Commitment) error {
```

Comment:
I think beside these checks, we need to ensure that the number of blobs per block must not larger than `MaxBlobsPerBlock = 6` you can add this config to `protocol_params.go` file

Reply:
You are right I forgot to check that, added